### DISCUSSION TITLE
Local examples do not match Github repository

### DISCUSSION THREAD

**Discussion by nguy** [tobac-project/tobac#231] [General]
After installing tobac via `mamba install tobac` (and sorting through the libraries still needed for examples I found that mostly things match up with examples. However, the first deviation is that I receive an error message that extrapolation is not yet implemented. So changing this variable to zero (in the OLR satellite example), yields the same number of trajectories. 

Secondly, the histograms at bottom of both the precip and OLR satellite examples are the same basic shape, but the counts are less in both. And in the case of the precipitation the lifetime values are not seen above 80 min. I'm guessing there has been some code update (they were last updated in Apr 2022). A very off the cuff guess without looking is a change in how things are grouped?

Anyways, wanted others to know in case they are trying to reproduce example results.

**Comment by w-k-jones** [comment]
Thanks for highlighting this, for reference, below is a comparison of the old output vs what I get for the latest hotfix patch for the satellite OLR tracking example:
Old:
![tobac_lifetime_dist_old](https://user-images.githubusercontent.com/22833757/214209598-b5ef63b9-e0d4-4dbd-be1b-0887ca65f972.png)
New:
![tobac_lifetime_dist_new](https://user-images.githubusercontent.com/22833757/214209593-b054f81f-fdec-4622-b6bb-36faf0d65fb6.png)

Note that some changes are expected, as part of the last release we found a number of underlying issues with trackpy that were causing spurious tracking output

We are currently looking into automating the production of the example notebooks, so hopefully this discrepancy shouldn't exist in future
